src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static class AzureAppConfigurationExtensions
         /// </summary>
         /// <param name="configurationBuilder">The configuration builder to add key-values to.</param>
         /// <param name="connectionString">The connection string used to connect to the configuration store.</param>
-        /// <param name="optional">If true, this method will not throw an exception if the configuration store cannot be accessed.</param>
+        /// <param name="optional">If true, and if an exception occurs while loading key-values from App Configuration store, this method will neither throw the exception nor load any key-values from App Configuration store.</param>
```

Comment:
If a configuration store cannot be accessed, isn't it implicit that the key-values will not be loaded?

Reply:
Sure, but we ignore key-values from App Config on other exceptions too (like KeyVaultReferenceException, RequestFailedException or OperationCanceledException) which can occur even if the store is accessible. So even though the exception may have occurred while processing a single key-value, we ignore all key-values from App Config.

Reply:
It seems to cover a bit too much to me.

I would go with "If true, any exceptions that occur while building the Azure App Configuration configuration provider will be suppressed and cause the provider to be empty. "

Let me know what you think.

Reply:
Here's another suggestion to keep it simple and avoid the customer's initial confusion.
`If true, any exception while loading key-values would be ignored.`

Reply:
Thanks for the suggestions. I like Jimmy's suggestion because its important to explicitly state that no key-values will be loaded for this provider. @abhilasharora, do you mind if we use that instead?

Reply:
Sure! I would change "Azure App Configuration configuration provider" to "Azure App Configuration provider". Also, it's not clear what provider being empty means. Is there another way to phrase it?

Reply:
How about this:
"If true, any exceptions that occur while building the App Configuration provider will be suppressed and no key-values will be loaded from App Configuration store."

Reply:
I don't think the sentence behaves the same way with that modification. Also, I would avoid the use of "key-values" and "App Configuration store" in favor of "settings" and "App Configuration instance".

This is very different from what I had before but, changing that previous sentence is hard.
"Determines the behavior of the App Configuration provider when an exception occurs. If false, the exception is thrown. If true, the exception is suppressed and no settings are populated from Azure App Configuration."